console/src/components/Services/ApiExplorer/ApiRequest/ApiRequest.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
 import jwt from 'jsonwebtoken';
 
 import TextAreaWithCopy from '../../../Common/TextAreaWithCopy/TextAreaWithCopy';
 import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
 import Tooltip from 'react-bootstrap/lib/Tooltip';
 import Modal from '../../../Common/Modal/Modal';
+import TooltipCustom from '../../../Common/Tooltip/Tooltip';
```

Comment:
As you are here, what about replacing bootstrap `Tooltip` and `OverlayTrigger` with our Tooltip component? It's used only in two places, it shouldn't be much work.